apps/site/src/scripts/component-demo-runtime.ts

```
@@ -0,0 +1,715 @@
+    if (loadedStyles.has(href)) return;
+    loadedStyles.add(href);
+    const link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.href = href;
+    document.head.appendChild(link);
+};
+
+const parseHtml = (html: string) => new DOMParser().parseFromString(html, 'text/html');
+
+const isUnsafeUrl = (value: string) => {
+    const normalized = value.replace(/[\u0000-\u001f\u007f\s]+/g, '').toLowerCase();
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00, \x1f.

(no-control-regex)